cardano-api/src/Cardano/Api.hs

```diff
@@ -101,21 +113,73 @@ shelleyGenSigningKey :: IO SigningKey
 shelleyGenSigningKey =
     SigningKeyShelley . Shelley.SKey <$> runSecureRandom genKeyDSIGN
 
+-- | Register a shelley staking pool.
+shelleyRegisterStakePool
+  :: ShelleyVerificationKeyHash
```

Comment:
:+1: yes, it's indeed the key hash. We should do the same for the stake address registration.